src/Prima/Pyxis/Form/Date.elm

```diff
@@ -275,9 +280,19 @@ renderDatePicker model dateModel =
             Html.text ""
 
 
-renderDatePickerIcon : String -> Html msg
-renderDatePickerIcon fieldId =
-    Html.label [ Attrs.class "form-input-group--datepicker__icon", Attrs.for fieldId ] []
+renderDatePickerIcon : Date model msg -> Html msg
+renderDatePickerIcon dateModel =
+    let
+        { id, onIconClick } =
+            computeOptions dateModel
+
+        iconAttrs : List (Html.Attribute msg)
+        iconAttrs =
+            Attrs.class "form-input-group--datepicker__icon"
+                :: ME.unwrap [] (Attrs.for >> List.singleton) id
+                ++ ME.unwrap [] (Events.onClick >> List.singleton) onIconClick
+    in
+    Html.label iconAttrs []
```

Comment:
Credo che il fatto che questo tag sia un `<label for="xxx">` sia un problema. 
Dovrebbe essere un `<i>`, anche perchè altrimenti ogni click sull'icona triggera il focus sul campo `Date`.

Reply:
Lo correggo, a maggior raggione perché non segue quel comportamento che invece apparentemente dovrebbe avere. La modifica parte proprio dal fatto che cliccando sull'icona non succede niente.